lib/irc/IrcServer.js

```diff
@@ -275,13 +275,12 @@ IrcServer.prototype.getAliasRegex = function() {
         {
             "$CHANNEL": ".*"  // the nick is unknown, so replace with a wildcard
         },
-        // The regex applies to the entire alias, so add a wildcard after : to
-        // match all domains.
-        ":.*"
+        // Only match the domain of the HS
+        ":" + hsdomain
```

Comment:
This is returning a regular expression. Domains unsurprisingly include `.`s. We should escape them.